kolibri/plugins/pdf_viewer/assets/src/views/PdfRendererIndex.vue

```diff
@@ -197,17 +200,14 @@
       this.currentLocation = this.savedLocation;
       this.showControls = true; // Ensures it shows on load even if we're scrolled
       const loadPdfPromise = PDFJSLib.getDocument(this.defaultFile.storage_url);
-
```

Comment:
Looks like your IDE might be applying more stringent linting standards than ours - blank lines in code can be good for readability, so would be better not to remove them.